easybuild/easyconfigs/k/koopmans/koopmans-1.1.0-foss-2024a.eb

```
@@ -0,0 +1,67 @@
+easyblock = 'PythonBundle'
+
+name = 'koopmans'
+version = '1.1.0'
+
+homepage = 'https://koopmans-functionals.org'
+description = """
+koopmans is a package for performing Koopmans spectral functional
+calculations with Quantum ESPRESSO, developed by researchers in the
+THEOS research group at EPFL.
+"""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Python-bundle-PyPI', '2024.06'),
+    ('matplotlib', '3.9.2'),
+    ('pydantic', '2.9.1'),
+    ('PyYAML', '6.0.2'),
+    ('scikit-learn', '1.5.2'),
+    ('SciPy-bundle', '2024.05'),
+    ('spglib-python', '2.5.0'),
+    ('typing-extensions', '4.11.0'),
+    ('koopmans-kcp', '0.2.0'),
+    ('koopmans-qe-utils', '0.1.0'),
+    ('QuantumESPRESSO', '7.4', '-minimal', ('foss', '2024a')),
```

```suggestion
    ('QuantumESPRESSO', '7.4', '-minimal'),
```
Not sure why this was like so? Definitely should be needed, and would cause problems for hierarchical module naming schemes.

---

Just me not knowing better, commited suggested changes.